Classes/ViewHelpers/ImageViewHelper.php

```diff
@@ -57,14 +60,12 @@ public function initializeArguments()
      */
     public function render($hash, $size = null)
     {
-        $identicon = $this->identiconFactory->create($hash);
-
-        $thumbnailImage = $this->getThumbnailImage($identicon->getImage(), $size);
+        $identicon = $this->identiconFactory->create(new IdenticonConfiguration($hash, $size));
 
         $this->tag->addAttributes(array(
-            'width' => $thumbnailImage->getWidth(),
-            'height' => $thumbnailImage->getHeight(),
-            'src' => $this->resourcePublisher->getPersistentResourceWebUri($thumbnailImage->getResource()),
+            'width' => $size,
+            'height' => $size,
+            'src' => 'data:image/png;base64,' . base64_encode($identicon),
```

Comment:
The base64 encoding can be optional ? What do you think to introduce a new argument in the VH ? i'm fine it's enabled by default, but can be disabled if needed.